migration-to-v2.mdx

```diff
@@ -0,0 +1,294 @@
+)
+
+# 2. Skip evaluation creation - it's automatic now!
+# (No galtea.evaluations.create() needed)
+
+# 3. Add version_id parameter to evaluation_tasks.create()
+galtea.evaluation_tasks.create(
+    version_id=version.id,  # ✅ Add this line
+    metrics=["your_metric"],
+    test_case_id="YOUR_TEST_CASE_ID",
+    actual_output="Your output"
+)
+```
+
+### Complete Before/After Example
+
+<details>
+<summary>📜 **Before (SDK v1.x)** - Click to expand</summary>
+
+```python
+from galtea import Galtea
+
+galtea = Galtea(api_key="YOUR_API_KEY")
+
+# Create version
+version = galtea.versions.create(
+    name="v1.0",
+    product_id="YOUR_PRODUCT_ID"
+)
+
+# ❌ Step 1: Explicitly create evaluation
+evaluation = galtea.evaluations.create(
+    test_id="YOUR_TEST_ID",
+    version_id=version.id
+)
+
+# ❌ Step 2: Create tasks using evaluation_id
+galtea.evaluation_tasks.create(
+    evaluation_id=evaluation.id,
+    metrics=["accuracy_v1"],
+    test_case_id="YOUR_TEST_CASE_ID",
+    actual_output="Barcelona"
+)
+```
+</details>
+
+<details>
+<summary>🎉 **After (SDK v2.0)** - Click to expand</summary>
+
+```python
+from galtea import Galtea
+
+galtea = Galtea(api_key="YOUR_API_KEY")
+
+# Create version (unchanged)
+version = galtea.versions.create(
+    name="v2.0",
+    product_id="YOUR_PRODUCT_ID"
+)
+
+# ✅ Step 1: Create tasks directly with version_id
+# (Evaluation is created automatically!)
+galtea.evaluation_tasks.create(
+    version_id=version.id,
+    metrics=["accuracy_v1"],
+    test_case_id="YOUR_TEST_CASE_ID",
+    actual_output="Barcelona"
+)
+```
+</details>
+
+## How to Migrate Your Code
+
+Let's look at a typical v1.x workflow and how it translates to v2.0.
+
+### Old Workflow (SDK v1.x)
+
+In v1.x, you would first create an `Evaluation` object and then use its ID to create `EvaluationTask` objects:
+
+```python
+# SDK v1.x Example
+from galtea import Galtea
+
+galtea = Galtea(api_key="YOUR_API_KEY")
+
+YOUR_PRODUCT_ID = "your_product_id"
+YOUR_TEST_ID = "your_test_id"
+YOUR_TEST_CASE_ID = "your_test_case_id"
+YOUR_METRIC_NAME = "your_metric_name"
+
+# 1. Create a Version (the optional properties were wrapped in a dict)
+version = galtea.versions.create(
+    name="v2.0-example",
+    product_id=YOUR_PRODUCT_ID,
+    optional_properties={  # Optional properties can be set directly
+      "description": "Initial version with basic summarization capabilities",
+      "system_prompt": "You are a helpful assistant that summarizes text.",
+      "model_id": "YOUR_MODEL_ID"
+    }
+)
+print(f"Created Version: {version.id}")
+
+# 2. Explicitly create an Evaluation
+evaluation = galtea.evaluations.create(
+    test_id=YOUR_TEST_ID,
+    version_id=version.id
+)
+print(f"Created Evaluation: {evaluation.id}")
+
+# 3. Create Evaluation Tasks using evaluation_id
+# (Assuming your_product_function(input) returns the actual output)
+test_case = galtea.test_cases.get(YOUR_TEST_CASE_ID) # Fetch a test case
```

Comment:
you call a test case from an id, only to use it to call the id. Right? 

But more importantly I would try to simulate the user flow as much as possible here, calling a test with a name and launching the evaluation for all the test cases in a loop.